dbms/src/Interpreters/Users.cpp

```diff
@@ -120,11 +121,24 @@ User::User(const String & name_, const String & config_elem, const Poco::Util::A
             /// Read table properties
             for (const auto & table : table_names)
             {
-                const auto config_filter = config_database + "." + table + ".filter";
+                String table_name = table;
+                String config_filter = config_database + "." + table_name + ".filter";
+
+                /// FIXME: naming scheme can be spoiled by user.
+                if (table.rfind("table[", 0) == 0)
```

Comment:
Don't understand, why `rfind` instead of `find` and what if there is only one <table> element and how can it be spoiled by user?

Reply:
Will add a test to check a single element.
The comment about spoiler is obsolete - will remove it.
`rfind` is just copy-paste - there is no particular reason.